src/test/java/org/opensearch/ad/ml/ModelManagerTests.java

```diff
@@ -8,992 +8,1060 @@
-
-   @Test
-   public void getEmptyStateNotFullSamples() {
-       ModelState<EntityModel> state = MLUtil
-           .randomModelState(new RandomModelStateConfig.Builder().fullModel(false).sampleSize(numMinSamples - 1).build());
-       assertEquals(
-           new ThresholdingResult(0, 0, 0),
-           modelManager.getAnomalyResultForEntity(new double[] { -1 }, state, "", null, shingleSize)
-       );
-       assertEquals(numMinSamples, state.getModel().getSamples().size());
-   }
-
-   @Test
-   public void scoreSamples() {
-       ModelState<EntityModel> state = MLUtil.randomModelState(new RandomModelStateConfig.Builder().fullModel(true).build());
-       modelManager.getAnomalyResultForEntity(new double[] { -1 }, state, "", null, shingleSize);
-       assertEquals(0, state.getModel().getSamples().size());
-       assertEquals(now, state.getLastUsedTime());
-   }
-
-   public void getAnomalyResultForEntity_withTrcf() {
-       AnomalyDescriptor anomalyDescriptor = new AnomalyDescriptor(point, 0);
-       anomalyDescriptor.setRCFScore(2);
-       anomalyDescriptor.setDataConfidence(1);
-       anomalyDescriptor.setAnomalyGrade(1);
-       when(this.trcf.process(this.point, 0)).thenReturn(anomalyDescriptor);
-
-       ThresholdingResult result = modelManager
-           .getAnomalyResultForEntity(this.point, this.modelState, this.detectorId, null, this.shingleSize);
-       assertEquals(
-           new ThresholdingResult(
-               anomalyDescriptor.getAnomalyGrade(),
-               anomalyDescriptor.getDataConfidence(),
-               anomalyDescriptor.getRCFScore()
-           ),
-           result
-       );
-   }
-
-   @Test
-   public void score_with_trcf() {
-       AnomalyDescriptor anomalyDescriptor = new AnomalyDescriptor(point, 0);
-       anomalyDescriptor.setRCFScore(2);
-       anomalyDescriptor.setAnomalyGrade(1);
-       // input dimension is 5
-       anomalyDescriptor.setRelevantAttribution(new double[] { 0, 0, 0, 0, 0 });
-       RandomCutForest rcf = mock(RandomCutForest.class);
-       when(rcf.getShingleSize()).thenReturn(8);
-       when(rcf.getDimensions()).thenReturn(40);
-       when(this.trcf.getForest()).thenReturn(rcf);
-       when(this.trcf.process(this.point, 0)).thenReturn(anomalyDescriptor);
-       when(this.entityModel.getSamples()).thenReturn(new ArrayDeque<>(Arrays.asList(this.point)));
-
-       ThresholdingResult result = modelManager.score(this.point, this.detectorId, this.modelState);
-       assertEquals(
-           new ThresholdingResult(
-               anomalyDescriptor.getAnomalyGrade(),
-               anomalyDescriptor.getDataConfidence(),
-               anomalyDescriptor.getRCFScore(),
-               0,
-               0,
-               anomalyDescriptor.getRelevantAttribution(),
-               null,
-               null,
-               null,
-               0,
-               numTrees
-           ),
-           result
-       );
-   }
+    private ModelManager modelManager;
```

Comment:
I know that this whitespace was generated by Spotless but it makes it really difficult to see what actually changed.

In the future, it would be helpful to have two commits.  One "whitespace only" commit followed by any substantive changes.